openhsi/data.py

```diff
@@ -112,7 +112,7 @@ def show(self):
 
 class CameraProperties():
     """Save and load OpenHSI camera settings and calibration"""
-    def __init__(self, json_path:str = "../assets/cam_settings.json", pkl_path:str = "../assets/cam_calibration.pkl", print_settings=False, **kwargs):
+    def __init__(self, json_path:str = None, pkl_path:str = None, print_settings=False, **kwargs):
```

Comment:
if we do things, need to check all notebook/test specify files etc.

Reply:
These settings files are outdated. Tests are passing on my end and on Github.

Reply:
Nw, I was just thinking of errors that piped up when stuff moved to nbs folder.